package.json

```diff
@@ -182,7 +182,7 @@
   },
   "scripts": {
     "precommit": "lint-staged && yarn run test",
-    "lint-js": "eslint --ignore-path .gitignore --ignore-pattern \"!**/.*\" .",
+    "lint-js": "eslint --cache --ignore-path .gitignore --ignore-pattern \"!**/.*\" .",
     "lint-css": "stylelint \"src/**/*.{css,less,styl,scss,sass,sss}\"",
```

Comment:
We could do a similar thing here, but `time yarn lint-css` is already only 1.5ish seconds

docs: https://stylelint.io/user-guide/usage/options/#cache